package.json

```diff
@@ -5,7 +5,15 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "dev": "webpack-dev-server --mode development ./src/index.js --output ./dist/main.js",
-    "build": "webpack --mode production ./src/index.js --output ./dist/main.js"
+    "build": "webpack --mode production ./src/index.js --output ./dist/main.js",
+    "precommit": "NODE_ENV=production lint-staged"
```

Comment:
production mode?

Reply:
was fixed, now is used webpack-server for production mode